packages/orderbook/test/js_api/gui.test.ts

```
@@ -936,12 +947,12 @@ ${dotrain}`;
 		});
 
 		it('should throw error if given dotrain is different', async () => {
+			const gui = new DotrainOrderGui();
 			let testDotrain = `${guiConfig}
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'testDotrain' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)